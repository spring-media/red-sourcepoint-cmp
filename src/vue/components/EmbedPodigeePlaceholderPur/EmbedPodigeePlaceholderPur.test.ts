import { mount } from '@vue/test-utils';
import { EmbedPodigeePlaceholderPur } from './';

describe('EmbedPodigeePlaceholderPur', () => {
  it('should render without any errors', () => {
    expect(
      mount(EmbedPodigeePlaceholderPur, {
        propsData: { privacyManagerId: 12345, privacyManagerIdDenyTracking: 67891, vendorId: 'abc' },
      }).element,
    ).toMatchSnapshot();
  });
});
