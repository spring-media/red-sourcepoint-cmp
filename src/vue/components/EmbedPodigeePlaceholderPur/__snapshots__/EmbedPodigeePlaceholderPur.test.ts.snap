// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmbedPodigeePlaceholderPur should render without any errors 1`] = `
<div
  class="embed-placeholder__container"
  data-vendor-id="abc"
>
  <div
    class="embed-placeholder__header--pur"
  >
    <svg
      fill="none"
      height="80"
      viewBox="0 0 80 80"
      width="80"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M30.814 16.9352C28.1511 15.2406 24.6665 17.1535 24.6665 20.3099V59.6898C24.6665 62.8462 28.1511 64.759 30.814 63.0645L61.7554 43.3745C64.2254 41.8027 64.2254 38.197 61.7554 36.6252L30.814 16.9352Z"
        stroke="#CED4DA"
        stroke-width="4"
      />
    </svg>
  </div>
   
  <div
    class="embed-placeholder__headline"
  >
     An dieser Stelle findest du Inhalte aus Podigee 
  </div>
   
  <div
    class="embed-placeholder__description--pur"
  >
    
      Um eingebettete Inhalte anzuzeigen, ist deine widerrufliche Einwilligung in die Übermittlung und Verarbeitung
      von personenbezogenen Daten notwendig, da die Anbieter der eingebetteten Inhalte als Drittanbieter diese
      Einwilligung verlangen [In diesem Zusammenhang können auch Nutzungsprofile (u.a. auf Basis von Cookie-IDs)
      gebildet und angereichert werden, auch außerhalb des EWR]. Indem du den Schalter auf „an“ stellst, stimmst du
      diesen (jederzeit widerruflich) zu. Dies umfasst auch deine
      
    <abbr
      title="Die betreffenden Drittländer, insb. die USA, weisen im Zweifel nicht das Datenschutzniveau auf, das Sie unter der DSGVO genießen. Das kann Nachteile wie eine erschwerte Durchsetzung von Betroffenenrechten, eine fehlende Kontrolle der Weiterverarbeitung und Übermittlung der Daten oder Zugriffe auf die Daten durch staatliche Stellen, insb. Behörden der USA, zu Kontroll- und Überwachungszwecken bedeuten, ohne dass Ihnen Rechtsbehelfe dagegen zustehen. Dies liegt nicht in der Hand von Axel Springer, sondern bedarf einer Regelung auf Gesetzgebungsebene."
    >
      Einwilligung in die Übermittlung bestimmter personenbezogener Daten in Drittländer, u.a. die USA
    </abbr>
    , nach Art. 49 (1) (a) DSGVO. Mehr Informationen dazu findest du
      
    <span>
      <a
        class="embed-placeholder__text-link"
        href="#"
        rel="noopener noreferrer"
        title="Zu den Privatsphäre Einstellungen"
      >
        hier
      </a>
      
          . Du kannst deine Einwilligung jederzeit über den Schalter und über
          
      <a
        class="embed-placeholder__text-link"
        href="#"
        rel="noopener noreferrer"
        title="Tracking widerrufen"
      >
        Widerruf Tracking
      </a>
    </span>
    
      am Seitenende widerrufen.
    
  </div>
   
  <div
    class="embed-placeholder__actions"
  >
    <div
      class="cmp-input-switch__container"
    >
      <label
        class="cmp-input-switch__label"
        for="input-switch-0"
      >
        Podigee freigeben
      </label>
       
      <div
        class="cmp-input-switch"
      >
        <div
          class="cmp-input-switch--hidden-accessible"
        >
          <input
            id="input-switch-0"
            role="switch"
            type="checkbox"
          />
        </div>
         
        <span
          class="cmp-input-switch__slider"
        />
      </div>
    </div>
  </div>
</div>
`;
